packages/excalidraw/change.ts

```diff
@@ -1100,7 +1102,7 @@ export class ElementsChange implements Change<SceneElementsMap> {
     try {
       // TODO: #7348 refactor away mutations below, so that we couldn't end up in an incosistent state
       ElementsChange.redrawTextBoundingBoxes(nextElements, changedElements);
-      ElementsChange.redrawBoundArrows(nextElements, changedElements);
+      ElementsChange.redrawBoundArrows(nextElements, changedElements, scene);
```

Comment:
Could you add a history test case to have this use case covered? (I didn't find one, you can get inspired by the tests testing the redraw above)

Reply:
I'm happy to. Can you please give me a little bit more context on what exactly needs to be tested? I'm not sure what the "above" means for inspiration in this context. I can't find any tests calling `redrawTextBoundingBoxes()`, if that's what "above" means here.

Are you proposing a test to see if redrawBoundArrows is called when history actions are triggered?

Reply:
Not just called, but also to check that the resulting redrawn points have the expected values. I am guessing you added the redrawn to cover cases like these:

https://github.com/user-attachments/assets/d2bea1c3-a8a2-49ea-a1dd-10a9506953bd

In that case, the arrow is correctly redrawn based on the new location of the diamond-shaped element when applying history action ("redo"), so we should have a test case verifying this works.

If you have other sinplayer cases in mind, let's talk about those as well.

Reply:
> I can't find any tests calling redrawTextBoundingBoxes(), if that's what "above" means here.

Yeah since it's part of the history integration testing. One such test case which is checking that `redrawTextBoundingBoxes` results with correctly redrawn elements is `should redraw remotely added bound text when it's container is updated through the history`.

Reply:
Done!